src/main/resources/application.properties

```diff
@@ -5,5 +5,5 @@ spring.datasource.password=kodilla_password
 spring.jpa.database=mysql
 spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true

```

Comment:
Do czego Ci ta zmiana w konfiguracji?
globally_quoted_identifiers

Reply:
Bez tego taki bład:
 at com.kodilla.ecommercee.EcommerceeApplication.main(EcommerceeApplication.java:10) ~[classes/:na]
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'group (id bigint not null auto_increment, name varchar(255), primary key (id)) e' at line 1

Reply:
Group jest zarezerwowaną nazwą w mysql (cześć polecenia group by) zamiast to obchodzić po prostu unikajmy użycia tych nazw. Tutaj ich lista:
https://stackoverflow.com/questions/23446377/syntax-error-due-to-using-a-reserved-word-as-a-table-or-column-name-in-mysql